apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -54,25 +46,27 @@ const SizeCheckbox: React.FunctionComponent = () => {
 };
 
 const OtherCheckbox: React.FunctionComponent = () => {
-  const [isCheckedControlled1, setCheckedControlled1] = React.useState(false);
-  const onChangeControlled1 = React.useCallback((checked) => {
-    setCheckedControlled1(checked);
+  const [isChecked, setisChecked] = React.useState(false);
+
+  const toggleCheckedTrue = React.useCallback(() => {
```

Comment:
naming nit: setCheckedTrue/setCheckedFalse instead of toggle? "Toggle" implies that it will flip the value to be the opposite of what it currently is (as in set to true if false or false if true) instead of to a specific value (always true or always false)

Reply:
yup , make sense. 
changed setCheckedTrue and setCheckedFalse.

Reply:
Changes completed and approved , hence resolving.